.github/codeql/codeql-config.yml

```diff
@@ -8,4 +8,3 @@ paths-ignore:
   - '**/Tests/'
   - '**/dist-history/'
   - '**/rollup.config.js'
-  - '/extensions/applicationinsights-angularplugin-js/'
```

Comment:
Is this one causing issues?

Reply:
An updated version of GitHub CoreQL started barfing on this because this folder only contains a readme.md